src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/Infrastructure/Startup.cs

```diff
@@ -41,10 +42,16 @@ public override void Configure(IFunctionsHostBuilder builder)
                 configuration.LogDiagnostics();
 
                 configuration.AdvancedConfiguration.Conventions()
+                    .DefiningMessagesAs(t => t.Namespace?.StartsWith("SFA.DAS.Apprentice.LoginService.Messages") == true ||
+                                             t.Name == nameof(ForceAutoEventSubscription))
+
                     .DefiningEventsAs(t => t.Namespace?.StartsWith("SFA.DAS.CommitmentsV2.Messages.Events") == true);
 
                 configuration.Transport.SubscriptionRuleNamingConvention(AzureQueueNameShortener.Shorten);
 
+                // when a SendInvitationCommand event is fired, route it to the login service queue
+                configuration.Transport.Routing().RouteToEndpoint(typeof(ls.SendInvitation), "sfa.das.apprenticeloginservice");
```

Comment:
I think there's a static class that contains the queue names, maybe we could move the string into that?

Reply:
Removed namespace alias and moved the queue name the the static class